samples: radio_test: Use MPSL output power splitter (#9391)
nrfconnect/sdk-nrf: samples/peripheral/radio_test/README.rst

   * - transmit_pattern
     - <sub_cmd>
     - Set transmission pattern.
   * - total_output_power
     - <tx output power>
     - Set total output power in dBm.
       This value includes SoC output power and front-end module gain.

Tx output power
===============

> This sample has a few different command that you can use to test device output power.
> The commands behavior varies depending on hardware configuration and Kconfig options.

Reviewer: ```suggestion
This sample has a few commands that you can use to test the device output power.
The behavior of the commands vary depending on the hardware configuration and Kconfig options as follows:
```

> 1. Radio Test without front-end module support:
>    The ``output_power`` command sets SoC output command with subcommands set.
>    The output power is set directly in the RADIO peripheral.

Reviewer: ```suggestion
* Radio Test without front-end module support:
  * The ``output_power`` command sets the SoC output command with a subcommand set.
    The output power is set directly in the radio peripheral.

```
Changed to bullet list, because this is not a sequence of steps.

> #. Radio Test with front-end module support in default configuration (the :kconfig:option:`CONFIG_RADIO_TEST_POWER_CONTROL_AUTOMATIC` Kconfig option is enabled).
>    The ``output_power`` command sets total output power including front-end module gain.
>    The ``total_output_power`` command sets total output power including front-end module gain with value in dBm unit provided by user.
>    For these commands the RADIO peripheral and FEM gain is calculated and set automatically to meet your requirements.
>    If it is not possible to set exact output power value, then the less possible value is set.

Reviewer: ```suggestion
* Radio Test with front-end module support in default configuration (the :kconfig:option:`CONFIG_RADIO_TEST_POWER_CONTROL_AUTOMATIC` Kconfig option is enabled):
  * The ``output_power`` command sets the total output power, including front-end module gain.
  * The ``total_output_power`` command sets the total output power, including front-end module gain with a value in dBm unit provided by user.
  * For these commands, the radio peripheral and FEM gain is calculated and set automatically to meet your requirements.
  * If it is not possible to set an exact output power value, a lower value is set.
  
```
Is the last point a user action? -> Can we use active voice like: "If you cannot set an exact value..."

Author: Last point is not an user action. It is done automatically so real output power can be different than requested by user. But this is done without any action from user

Reviewer: Maybe then something like: "If an exact output power value cannot be set, a lower value is used.

Author: Done

> #. Radio Test with front-end module support and disabled the kconfig:option:`CONFIG_RADIO_TEST_POWER_CONTROL_AUTOMATIC` Kconfig option.
>    The ``output_power`` command sets SoC output command with subcommands set.
>    The ``fem`` command with the ``tx_gain`` subcommand sets the front-end module gain in arbitrary value for given front-end module.
>    This configurations can be used to perform tests of your hardware design.

Reviewer: ```suggestion
* Radio Test with front-end module support and the kconfig:option:`CONFIG_RADIO_TEST_POWER_CONTROL_AUTOMATIC` Kconfig option disabled:
  * The ``output_power`` command sets the SoC output command with a subcommands set.
  * The ``fem`` command with the ``tx_gain`` subcommand sets the front-end module gain to an arbitrary value for given front-end module.
  * You can use this configuration to perform tests on your hardware design.
```
